lm_eval/tasks/med_concepts_qa/med_concepts_qa_atc_hard.yaml

```
@@ -0,0 +1,6 @@
+dataset_name: atc_hard
+group: med_concepts_qa_atc
+group_alias: atc
```

```suggestion
group: med_concepts_qa_atc
group_alias: atc
```
as described over DM, these 2 lines for each config file need changing

---

Done